usecases/stable/documentationcamera/recording.documentationcamera.realtime/fileserver/Dockerfile

```diff
@@ -15,7 +15,7 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
-# Date: 2016-09-01
+# Date: 2016-09-09
 
 FROM seresearch/opendlv-core-on-opendavinci-ubuntu-16.04-complete:latest
```

Comment:
We still have :latest as tag here even though that this Dockerfile is residing in stable/.

According to our process, this is not allowed.